docs/flocker-features/pure-storage-configuration.rst

```diff
@@ -0,0 +1,13 @@
+.. _pure-storage-backend:
+
+=================================
+Pure Storage Block Device Backend
+=================================
+
+.. begin-body
+
+Pure Storage provides a Flocker backend plugin.
+
+For more information, including installation, testing and usage instructions, visit the `Pure Storage Flocker Driver on GitHub <https://github.com/sdodsley/purestorage-flocker-driver>`_
```

Comment:
Thanks @patrick-east,
I updated the link.
Will you update the Flocker documentation link on your repo:

> Install ClusterHQ/Flocker
> https://docs.clusterhq.com/en/0.3.2/gettingstarted/index.html

Should link to the latest stable version of Flocker, which is currently:
 * https://docs.clusterhq.com/en/1.12.0/

And this week we're hoping to release 1.13.0 so it'd be great if you could test the driver against Flocker master builds:
 * http://build.clusterhq.com/results/omnibus/master/